fuzz/fuzz_targets/streams.rs

```diff
@@ -0,0 +1,71 @@
+#![no_main]
+
+use arbitrary::{Arbitrary, Unstructured};
+use libfuzzer_sys::fuzz_target;
+
+extern crate proto;
+use proto::fuzzing::{ResetStream, Streams};
+use proto::{Dir, Side, StreamId, VarInt};
+
+#[derive(Arbitrary, Debug)]
+struct StreamParams {
+    side: Side,
+    max_remote_uni: u16,
+    max_remote_bi: u16,
+    send_window: u16,
+    receive_window: u16,
+    stream_receive_window: u16,
+    dir: Dir,
+}
+
+#[derive(Arbitrary, Debug)]
+enum Operation<A>
+where
+    A: Arbitrary,
+{
+    Open(Vec<A>),
+    Accept(Dir),
+    Finish(StreamId),
+    ReceivedStopSending((StreamId, VarInt)),
```

Comment:
```suggestion
    ReceivedStopSending(StreamId, VarInt),
```
would be a bit more conventional style--unless I'm missing the motivation here?

Reply:
:+1: no motivation just poor style :-) corrected.

Reply:
> This looks almost ready to me! Is there anything else you wanted to cover before merging?

Very good! Thank you for the help and for making `quinn` hospitable to new comers.